README.md

```diff
@@ -1,5 +1,15 @@
 # Timesheetsdb
-Timesheets web application
+Time sheets database for web application
+
+
+## Prerequisites
+
+```
+PostgreSQL v13.0
+NodeJS v14.13.1
+npm v6.14.8
+
+```
 
```

Comment:
Maybe add a `Database` section - should have a pointer to a decent Postgres install for linux.
Also say that a database is needed (suggest `timesheets` with a user account that e.g. `timesheets_db_user` that must match whatever is given in the `.env` file below.

Then just give a simple list of the tables created.